sdk/storage/Azure.Storage.Blobs/src/Models/BlobProperties.cs

```diff
@@ -206,8 +206,8 @@ public partial class BlobProperties
         /// <summary>
         /// The tier of page blob on a premium storage account or tier of block blob on blob storage LRS accounts.
         /// For a list of allowed premium page blob tiers, see
-        /// https://docs.microsoft.com/en-us/azure/virtual-machines/windows/premium-storage#features. For blob
-        /// storage LRS accounts, valid values are Hot/Cool/Archive.
+        /// https://docs.microsoft.com/en-us/azure/virtual-machines/windows/premium-storage#features. For general 
```

Comment:
I think if you remove the trailing space after `general`. The CI will start running green.